xrdb.md

```diff
@@ -0,0 +1,16 @@
+# xrdb
+
+> X window server's resource database utility for Unix-like systems.
+> More information: https://www.x.org/releases/X11R7.7/doc/man/man1/xrdb.1.xhtml
+
+- Run the utility in interactive mode:
+
+ `xrdb`
```

Comment:
There is still a space here which should be. Please apply these suggestions from here rather than just editing the page yourself.

```suggestion
`xrdb`
```